packages/dx-react-grid/src/types/data-type-provider/data-type-provider.types.ts

```diff
@@ -24,6 +24,15 @@ export namespace DataTypeProvider {
     onValueChange: (newValue: any) => void;
     /** "true" if users should not be able to edit the value, "false" otherwise. */
     disabled: boolean;
+    /** "true" if editor should be focused on render, "false" otherwise. */
+    autoFocus: boolean;
+    /** Switches the editor from edit state when it loses focus (used for Inline Cell Editing). */
+    onBlur: () => void;
```

Comment:
This action call the `onCommitChanges` function. See our discussion about it https://trello.com/c/DQIUWHI4/1701-1-add-note-about-datatypeprovider-editor-props-with-inline-cell-editing-mode

Reply:
@RomanTsukanov 
Re-check this, please (also changed in `packages/dx-react-grid/docs/reference/data-type-provider.md`)